src/item/store/actions.ts

```diff
@@ -0,0 +1,61 @@
+import { Avo } from '@viaa/avo2-types';
+import queryString from 'query-string';
+import { Action, Dispatch } from 'redux';
+
+import {
+	ItemActionTypes,
+	SetItemErrorAction,
+	SetItemLoadingAction,
+	SetItemSuccessAction,
+} from './types';
+
+const getItem = (id: string) => {
+	return async (dispatch: Dispatch, getState: any): Promise<Action | null> => {
+		const { item } = getState();
+
+		// don't fetch a item we already have in store
+		if (item[id]) {
+			return null;
+		}
+
+		dispatch(setItemLoading(id));
+
+		try {
+			const response = await fetch(
+				`${process.env.REACT_APP_PROXY_URL}/item?${queryString.stringify({ id })}`,
+				{
+					method: 'GET',
+					headers: {
+						'Content-Type': 'application/json',
+					},
+				}
+			);
+
+			const data = await response.json();
+
+			return dispatch(setItemSuccess(id, data as Avo.Item.Response));
+		} catch (e) {
+			return dispatch(setItemError(id));
+		}
+	};
+};
+
+const setItemSuccess = (id: string, data: Avo.Item.Response): SetItemSuccessAction => ({
+	id,
+	data,
+	type: ItemActionTypes.SET_DETAIL_SUCCESS,
+});
+
+const setItemError = (id: string): SetItemErrorAction => ({
```

Comment:
waar wordt dit weergegeven aan de gebruiker?

Reply:
Geen idee, ik heb dat overgenomen van uit de code die jij al had voor het fetchen van die items.
Die toast heeft niks met opzet ven redux store te maken.

Reply:
bij mij stond er een console log, zodat we weten dat dat nog moet gebeuren.
https://github.com/viaacode/avo2-client/pull/22/files#diff-65c9226a5bcda69196e39cbf8947e229L63

Mss ook ergens een todo zetten in jouw code?
maar het is wat vreemd dat we een tijdelijke toast moeten binden aan permanente state.

Reply:
Todo's horen niet in code terug, als er iets niet meteen geïmplementeerd kan worden omdat het blocked is door een dependency mag dat als ticket op de backlog in JIRA toegevoegd worden.

Reply:
Zet dan een console.error in de plaats
Nu worden die errors gewoon ingeslikt

Reply:
Added in backlog. https://district01.atlassian.net/browse/AVO2-381